youtube_dl/extractor/dplay.py

```diff
@@ -296,6 +296,17 @@ def _real_extract(self, url):
 
         title = remove_end(self._og_search_title(webpage), ' | Dplay')
 
+        season_number = episode_number = series = None
+        videodata = self._html_search_regex(
+            r'\sdigitalData\s?=\s?JSON\.parse\([\'"]([^;\n]+)[\'"]\)',
```

Comment:
If you assume to capture a dict then capture curly braces. Whitespace must be `\s+`.